pkg/gatewayserver/io/basicstationlns/basicstationlns_test.go

```diff
@@ -52,13 +57,14 @@ var (
 	timeout = 10 * test.Delay
 )
 
+func eui64Ptr(eui types.EUI64) *types.EUI64 { return &eui }
+
 func TestAuthentication(t *testing.T) {
 	// TODO: Test authentication. We're gonna provision authentication tokens, which may be API keys.
```

Comment:
Will this be added to this PR or are we doing this in the next?

Reply:
The Auth Header check will be in the follow up PR. Client-side TLS is for later #558